File: messagerie/src/main/java/com/rh/messagerie/dto/MessageRequest.java
package com.rh.messagerie.dto;

import com.rh.messagerie.entities.File;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;
import java.util.List;

@AllArgsConstructor
@Data
public class MessageRequest {
    String text;
    Long sender;
    Long receiver;
}

File: rh-front/src/app/models/messageRequest.ts
import { File } from "./file";

export class MessageRequest {
    text: string;
    sender: number;
    receiver: number;
    files?: File[];
    constructor(
        sender: number,
        receiver: number,
        text: string,
        files: File[]
    ) {
        this.text = text;
        this.sender = sender;
        this.receiver = receiver;
        this.files = files;
    }
}